frontend/src/components/navbar/NavbarSearchPanel.vue

```diff
@@ -19,9 +19,12 @@
             </div>
         </div>
         <div class="navbar-item is-paddingless navbar-search-btn">
-            <span class="icon is-large">
-                <i class="fas fa-lg fa-search" />
-            </span>
+            <button class="button">
+                <span class="icon is-large">
+                    <i class="fas fa-lg fa-search" />
+                </span>
+                <span v-if="windowWidth < 1088">Search</span>
```

Comment:
create computed prop for `windowWidth < 1088` or even better you can add specific css classes and hide/show span using css just